### PR TITLE
No one wants a FactoryFactory

### PR DESCRIPTION
**Pull request by jblebrun**

### CODE REVIEW

**Comment by shans** [comment]
Nice cleanup!

File: java/arcs/android/storage/handle/AndroidHandleManager.kt [hunk 1]
```diff
@@ -9,37 +9,16 @@
  * http://polymer.github.io/PATENTS.txt
  */
 package arcs.android.storage.handle
-
```
Comment (csilvestrini) [inline java/arcs/android/storage/handle/AndroidHandleManager.kt]: I think we like this blank line

Comment (jblebrun) [reply java/arcs/android/storage/handle/AndroidHandleManager.kt]: Not sure how that happened!

**Comment by jblebrun** [comment]
Breakage in g3. However, the good news is I can fix it independently.

**Comment by jblebrun** [comment]
(cl/301521874)

**Pull request merged**